centos/SPECS/softethervpn.spec

```diff
@@ -1,7 +1,7 @@
 %define majorversion 4
-%define minorversion 19
-%define buildversion 9582
-%define dateversion 2015.10.06
+%define minorversion 23
```

Comment:
Why 23? 24 has already been released

Reply:
according to your changelog file on the root directory 
https://github.com/SoftEtherVPN/SoftEtherVPN/blob/master/ChangeLog